WordPress/Classes/ViewRelated/Me/MyProfileViewController.swift

```diff
@@ -0,0 +1,137 @@
+import UIKit
+import WordPressShared
+
+class MyProfileViewController: UITableViewController {
+    var account: WPAccount! {
+        didSet {
+            self.service = AccountSettingsService(accountID: account.userID.integerValue, api: account.restApi)
+        }
+    }
+
+    var service: AccountSettingsService! {
+        didSet {
+            subscribeSettings()
+        }
+    }
+
+    var settingsSubscription: AccountSettingsSubscription?
+
+    var handler: ImmuTableViewHandler!
+
+    // MARK: - Table View Controller
+
+    required convenience init() {
+        self.init(style: .Grouped)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.title = NSLocalizedString("My Profile", comment: "My Profile view title")
+
+        ImmuTable.registerRows([
+            EditableTextRow.self
+            ], tableView: self.tableView)
+
+        handler = ImmuTableViewHandler(takeOver: self)
+
+        WPStyleGuide.resetReadableMarginsForTableView(tableView)
+        WPStyleGuide.configureColorsForView(view, andTableView: tableView)
```

Comment:
Nothing wrong with the code, but are you aware that view and tableView are the same objects on a UITableViewController?